README.md

```diff
@@ -1,39 +1,35 @@
-<p align="center">
-  <img alt="Nimble logo" src="https://assets.nimblehq.co/logo/light/logo-light-text-320.png" />
-</p>
+# GitHub Actions Runner for Android
 
----
+The repository for proof of concept of [Custom Runner for Github Actions (Android) initiative](https://www.notion.so/nimblehq/Custom-Runner-for-Github-Actions-Android-bb4c0b6310184e44afe9b3bea0989a57?d=72eb950e-33be-482b-8e17-40c92023073d)
 
-A collection of templates:
+## Setup custom runner
 
-* **[CoroutineTemplate](https://github.com/nimblehq/android-templates/tree/kotlin/CoroutineTemplate)**
-* **[RxJavaTemplate](https://github.com/nimblehq/android-templates/tree/kotlin/RxJavaTemplate)**
+### Prerequisites
 
-## Setup
+- Machine running macOS from [MacStadium](https://www.macstadium.com/) or a physical one
+- Android Studio installation is recommended as it will help to install all the required Java, Android
+  SDK, also the AVD which needed for running UI test
 
-1. Clone or download this repository to your local machine, then extract and open the folder
-2. Run `newproject.sh` script to create a new project with the following inputs:
+### Add custom (self-hosted) runner
 
-```
-  -h, --help                         Display this usage message and exit
-  -t, --template [TEMPLATE]          Select template: "rx" - RxJavaTemplate or "crt" - CoroutineTemplate (i.e. rx)
-  -p, --package-name [PACKAGE_NAME]  New package name (i.e. com.example.package)
-  -n, --app-name [APP_NAME]          New app name (i.e. MyApp, "My App")
-```
+1. Open the **Settings** menu of the repository. Go to **Actions** -> **Runners** -> click on the **Add runner** button
+2. Make sure the Operating System dropdown shows **macOS** as a selected item, then run all of the below commands in your machine in order. Specify configurations when prompted.
+3. (Optional) In the directory where the runner was installed, run `./svc.sh install`. This program is for help checking the status of the runner.
+4. In the directory where the runner was installed, run `./run.sh` to open the runner to listening for the upcoming jobs.
```

Comment:
Same here.

4. In the directory where the runner was installed, run 

   ```shell
   ./run.sh
   ```
    
   to open the runner to listening for the upcoming jobs.